### PR TITLE
show analysis progress

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds analysis progress in the status cases command

**How to test**:
1. install on stage of the hasta machine: `bash servers/resources/c.s.se/update-cg-stage.sh thisbranch`
1. activate stage: `us`
1. start an analysis cg analysis -f vitalmouse
1. run trailblazer scan to pick up the new analysis run
1. run following command: `cg status cases -o count`

**Expected outcome**:
`cg` should be providing you with a progress report of your vitalmouse analysis making. 
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @ingkebil 
- [x] tests executed by @patrikgrenfeldt 
- [x] "Merge and deploy" approved by @ingkebil 
Thanks for filling in who performed the code review and the test!

This is minor **version bump** because it adds new but compatible functionality

### CODE REVIEW

**Comment by ingkebil** [comment]
Have you seen that the coverage drops 1.2% when we pull this PR in? Anything we can do about that?

**Comment by patrikgrenfeldt** [comment]
> 
> 
> Have you seen that the coverage drops 1.2% when we pull this PR in? Anything we can do about that?

Sure, with your context injection technique :)

**Comment by patrikgrenfeldt** [comment]
image
👍

File: cg/cli/status.py [hunk 1]
```diff
@@ -95,7 +97,8 @@ def present_string(case, param, show_negative):
 @click.option('--days', default=31, help='days to go back')
 @click.option('--internal-id', help='search by internal id')
 @click.option('--name', help='search by name given by customer')
-@click.option('--action', type=click.Choice(FAMILY_ACTIONS), help='filter by action')
+@click.option('--case-action', type=click.Choice(FAMILY_ACTIONS), help='filter by case action')
+@click.option('--progress-status', help='filter by progress status')
```
Comment (patrikgrenfeldt) [inline cg/cli/status.py]: add click-choice

File: tests/cli/test_cli_status_cases.py [hunk 2]
```diff
@@ -1,123 +1,121 @@
...
-        ['--database', disk_store.uri, 'status', 'cases'])
+    result = cli_runner.invoke(cases, obj=base_context)
 
     # THEN the family should be listed
     assert result.exit_code == 0
     assert family.internal_id in result.output
 
 
-def ensure_application_version(disk_store, application_tag='dummy_tag'):
+def ensure_application_version(base_store, application_tag='dummy_tag'):
```
Comment (patrikgrenfeldt) [inline tests/cli/test_cli_status_cases.py]: rename base_store to store

Comment (patrikgrenfeldt) [reply tests/cli/test_cli_status_cases.py]: renamed

File: cg/cli/status.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/cli/status.py]: added

**Comment by patrikgrenfeldt** [comment]
deployed

**Pull request merged**